src/frontend/apps/standalone_site/src/features/ClassRooms/components/ClassRooms.tsx

```diff
@@ -0,0 +1,133 @@
+import { Text, Box, Pagination } from 'grommet';
+import { Alert } from 'grommet-icons';
+import { useClassrooms } from 'lib-classroom';
+import React, { Fragment, PropsWithChildren, ReactNode, useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { ReactComponent as ClassroomsIcon } from 'assets/svg/iko_webinairesvg.svg';
+import { WhiteCard, ContentCards } from 'components/Cards';
+import { ContentSpinner } from 'components/Spinner';
+
+import ClassRoom from './ClassRoom';
+
+const messages = defineMessages({
+  ClassroomTitle: {
+    defaultMessage: 'Classrooms',
+    description: 'Classrooms title',
+    id: 'features.ClassRooms.ClassroomTitle',
+  },
+  NoClassroom: {
+    defaultMessage: 'There is no classroom to display.',
+    description: 'Text when there is no classroom to display.',
+    id: 'features.ClassRooms.NoClassroom',
+  },
+  Error: {
+    defaultMessage: 'Sorry, an error has occurred.',
+    description: 'Text when there is an error.',
+    id: 'features.ClassRooms.Error',
+  },
+});
+
+const BoxPagination = styled(Box)`
+  & button {
+    color: #055fd2;
+    background-color: #f2f7fd;
+  }
+  & button[aria-current] {
+    background-color: #dbebff;
+  }
+  & button svg {
+    stroke: #055fd2;
+  }
+`;
+
+const ContainerInfo = ({ children }: PropsWithChildren<ReactNode>) => {
+  return (
+    <Box
+      direction="row"
+      align="center"
+      justify="center"
+      margin={{ top: 'medium' }}
+      gap="small"
+    >
+      {children}
+    </Box>
+  );
+};
+
+function ClassRooms() {
+  const intl = useIntl();
+  const nbItemPerPage = 20;
```

Comment:
IMO this should be shared with all resources. I think we would like to have the same limit when we list videos, webinars, classrooms, etc.